frontend/src/views/CellGuide/components/CellGuideCard/style.ts

```diff
@@ -28,16 +28,13 @@ export const CellGuideView = styled.div`
   column-gap: ${SIDEBAR_COLUMN_GAP_PX}px;
   margin: auto;
   max-width: 1440px;
-
   ${(props: CellGuideViewProps) => {
     const { skinnyMode } = props;
     const spaces = getSpaces(props);
     const space = skinnyMode ? spaces?.xl : spaces?.xxl;
 
     return `
-    padding: ${TOP_PADDING_PX}px ${space}px 0px
-      ${space}px;
-
+    padding: 0px ${space}px 0px ${space}px;
```

Comment:
i removed the top padding from `CellGuideView` and into `CellGuideCardHeader`, which will make sure that the padding sticks with the header as you scroll down but still shows up in the static view